### PR TITLE
Dark Mode : Connected account popover

### PR DESCRIPTION
**Pull request by GuillaumeRx**
## Explanation

Dark mode support for connected account popover

## More information

* Fixes: #13719

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]
Excellent!

File: ui/components/app/connected-accounts-permissions/index.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
   display: flex;
   flex-direction: column;
-  color: var(--Grey-500);
+  color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/connected-accounts-permissions/index.scss:6]: If this is text it should be 
```suggestion
  color: var(--color-text-alternative);
```

File: ui/components/ui/popover/index.scss [hunk 2]
```diff
@@ -100,7 +101,7 @@
   &-arrow {
     width: 22px;
     height: 22px;
-    background: white;
+    background: var(--color-background-default);
```
Comment (georgewrmarshall) [inline ui/components/ui/popover/index.scss:104]: What is this arrow and where does it show up? Doesn't look great on storybook?

Screen Shot 2022-03-15 at 12 55 28 PM

**Review by georgewrmarshall — COMMENTED** [review]
Looks good! Have a couple suggestions

File: ui/components/ui/popover/index.scss (hunk 2 above)
Comment (GuillaumeRx) [reply ui/components/ui/popover/index.scss:104]: I don't know the intended use of this but it seems to be not used.

Comment (georgewrmarshall) [reply ui/components/ui/popover/index.scss:104]: Ah k let's just updated and leave it 😄

**Review by darkwing — APPROVED** [review]

**Review by georgewrmarshall — APPROVED** [review]
lgtm

**Pull request merged**